src/maprules.jl

```diff
@@ -103,41 +118,45 @@ end
 
 ### Rules that don't need a neighborhood window ####################
 
-# optmap
+# broadcast_with_optimisation
 # Map kernel over the grid, specialising on PerformanceOpt.
 
 # Run kernel over the whole grid, cell by cell:
-function optmap(
+function broadcast_with_optimisation(
     f, simdata::AbstractSimData{S}, proc, ::NoOpt, ::Val{<:Rule}, rkeys
 ) where S<:Tuple{Y,X} where {Y,X}
-    procmap(proc, 1:X) do j
+    broacast_on_processor(proc, 1:X) do j
```

Comment:
Typo in several places ? (`broacast_on_processor` => `broadcast_on_processor`)

Reply:
Yeah this PR is under heavy edits, maybe review in a week or so...